src/node/desktop/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "clean": "ts-node scripts/clean.ts",
     "lint": "eslint ./src ./test",
-    "make": "yarn run clean && electron-forge make",
+    "make": "wrap () { set -e ; yarn run clean ; electron-forge make $*; ./scripts/rename-dmg.sh;}; wrap",
```

Comment:
What does `wrap` technique do here? Will this also work on Windows?

Also, invoking a shell script (`rename-dmg.sh`) will not work on Windows. So maybe call a script (js or ts) that checks for platform and does the right thing here.

Reply:
Wrap creates an inline bash function so every command inside it will have the same parameters that we give to `yarn make` for example `yarn make --clean` so it will get those parameters. There's a use for this, but probably related to fix library paths